packages/radio/src/react/button.js

```diff
@@ -1,24 +1,24 @@
-import { compose, css } from 'glamor'
+import { compose, css, checked } from 'glamor'
```

Comment:
`checked` coming from glamor is probably not what you want. it's always truthy so all buttons are shown as checked